### PR TITLE
GQL-96: Updating the README

### PR DESCRIPTION
**Pull request by mandyparson**
# Overview

### What is the feature?

Current README updated with information about CLI commands for SIT and UAT spin up

### What is the Solution?

Added updated CLIs

### What areas of the application does this impact?

Added Local Environment Spin-Up section and edited env variables table

# Testing

### Reproduction steps

- **Environment for testing: Local, SIT, and UAT
- **Collection to test with: N/A

1. Spin up each environment and check that it works in apollo studio.
2. Do this with MMT-3954 do double check it spins up correctly. 

### Attachments
Screenshot 2024-12-10 at 11 35 44 AM

# Checklist

- [ ] I have added automated tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,10 +40,20 @@ CMR-GraphQL uses a few environment variables for configuring runtime options:
 |MMT_ROOT_URL||URL to ping when retrieving metadata from MMT e.g. https://mmt.earthdata.nasa.gov|
 |DRAFT_MMT_ROOT_URL||URL to ping when retrieving draft metadata from Draft MMT e.g. https://draftmmt.earthdata.nasa.gov|
 |LAMBDA_TIMEOUT|30|Number of seconds to set the Lambda timeout to.|
-|EDL_KEY_ID, EDL_JWK, EDL_CLIENT_ID|For facilitating EDL connection -- obtain these from a dev|
+|EDL_KEY_ID, EDL_JWK, EDL_CLIENT_ID, EDL_PASSWORD| .bash or .zshrc file|For facilitating EDL connection -- obtain these from a dev and upload. After making changes, remember to run $ source ~/.zshrc|
 
-### Serverless Framework
+### Local Environment Spin-Up
+If using cmr-graphql in conjunction with MMT, setup local CMR database with the [instructions found here](https://github.com/nasa/mmt/tree/main). Then utilize one of the following commands depending on the
```
Comment (macrouch) [inline README.md:46]: MMT specific help should exist in the MMT readme. Adding this information here will only add confusion for people who aren't developing MMT.

**Review by cgokey — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -40,10 +40,20 @@ CMR-GraphQL uses a few environment variables for configuring runtime options:
...
+|EDL_KEY_ID, EDL_JWK, EDL_CLIENT_ID, EDL_PASSWORD| .bash or .zshrc file|For facilitating EDL connection -- obtain these from a dev and upload. After making changes, remember to run $ source ~/.zshrc|
 
-### Serverless Framework
+### Local Environment Spin-Up
+If using cmr-graphql in conjunction with MMT, setup local CMR database with the [instructions found here](https://github.com/nasa/mmt/tree/main). Then utilize one of the following commands depending on the 
+environment you wish to point graphql towards. You do not need quotations around any variables EXCEPT for UAT's EDL_PASSWORD.
+
+|Environment|CLI|
+|-|-|
+|Local| $ CMR_ROOT_URL=http://localhost:4000 EDL_CLIENT_ID=SIT Client ID EDL_PASSWORD=SIT Password npm start|
```
Comment (macrouch) [inline README.md:51]: My commands I use look very different from these. These actually wouldn't work for me without changing my environment. We already describe the environment variables needed, that should be enough for a developer to understand, and they can use their personal preference on how to save those variables.

File: README.md [hunk 3]
```diff
@@ -40,10 +40,20 @@ CMR-GraphQL uses a few environment variables for configuring runtime options:
 |MMT_ROOT_URL||URL to ping when retrieving metadata from MMT e.g. https://mmt.earthdata.nasa.gov|
 |DRAFT_MMT_ROOT_URL||URL to ping when retrieving draft metadata from Draft MMT e.g. https://draftmmt.earthdata.nasa.gov|
 |LAMBDA_TIMEOUT|30|Number of seconds to set the Lambda timeout to.|
-|EDL_KEY_ID, EDL_JWK, EDL_CLIENT_ID|For facilitating EDL connection -- obtain these from a dev|
+|EDL_KEY_ID, EDL_JWK, EDL_CLIENT_ID, EDL_PASSWORD| .bash or .zshrc file|For facilitating EDL connection -- obtain these from a dev and upload. After making changes, remember to run $ source ~/.zshrc|
```
Comment (macrouch) [inline README.md:43]: `.bash or .zshrc file` is not a default value that is used, so it doesn't make sense in that column. And again, there are more than one valid ways to save environment variables, adding that you have to run `$ source ~/.zshrc` is not valid for all users, and we should leave it up to the dev to set variables however they want to

**Review by macrouch — CHANGES_REQUESTED** [review]

**Comment by mandyparson** [comment]
Not needed

**Pull request closed without merging**